src/nspawn/nspawn-seccomp.h

```diff
@@ -21,4 +21,6 @@
 
 #include <sys/types.h>
 
-int setup_seccomp(uint64_t cap_list_retain);
+#include "seccomp-util.h"
```

Comment:
why include that here? you don't need any types of it here? usually we try to keep header inclusions as local as possible